theme_avantgarde/views/customizations.xml

```diff
@@ -1,60 +1,155 @@
+        <attribute name="class" add="m-1 m-md-0" separator=" "/>
+    </xpath>
+    <xpath expr="//div[hasclass('row')]/div/div" position="attributes">
+        <attribute name="class" add="shadow border" separator=" "/>
+        <attribute name="style">border-color: rgb(11, 13, 99) !important; box-shadow: rgb(11, 13, 99) -25px -25px 0px 0px !important; border-width: 8px !important;</attribute>
+    </xpath>
+    <xpath expr="//h3" position="replace" mode="inner">
+        <span style="font-weight: bolder;">Excellence</span>
+    </xpath>
+    <xpath expr="(//p[hasclass('card-text')])[1]" position="replace" mode="inner">
+We conduct state-of-the-art research and development to solve complex design challenges, taking the latest advances out of the lab and into the hands of architects and engineers.</xpath>
+    <xpath expr="//div[hasclass('row')]/div[2]/div" position="attributes">
+        <attribute name="class" add="shadow border" separator=" "/>
+        <attribute name="style">border-color: rgb(255, 255, 255) !important; border-width: 8px !important; box-shadow: rgb(78, 77, 77) 0px 0px 100px -40px !important;</attribute>
+    </xpath>
+    <xpath expr="(//h3)[2]" position="replace" mode="inner">
+        <span style="font-weight: bolder;">Collaboration</span>
+    </xpath>
+    <xpath expr="(//p[hasclass('card-text')])[2]" position="replace" mode="inner">
+        We thrive on rich collaborations to push our thinking. A continuous state of reinvention, driven by our partners in the process, is essential to our work.
+    </xpath>
+    <xpath expr="//div[hasclass('row')]/div[3]/div" position="attributes">
+        <attribute name="class" add="shadow border" separator=" "/>
+        <attribute name="style">box-shadow: rgb(240, 142, 128) 25px 25px 0px 0px !important; border-color: rgb(240, 142, 128) !important; border-width: 8px !important;</attribute>
+    </xpath>
+    <xpath expr="(//h3)[3]" position="replace" mode="inner">
+        <span style="font-weight: bolder;">Sustainability</span>
+    </xpath>
+    <xpath expr="(//p[hasclass('card-text')])[3]" position="replace" mode="inner">
+        Sustainability is at the heart of our design approach. We audit projects against global standards as well as our own, more comprehensive, responsibility framework.
     </xpath>
 </template>
 
 <!-- ======== TEXT-IMAGE ======== -->
  <template id="s_text_image" inherit_id="website.s_text_image" name="Avantgarde s_text_image">
+    <xpath expr="//section" position="attributes">
+        <attribute name="class" add="o_cc o_cc3 o_full_screen_height" separator=" "/>
+    </xpath>
+    <xpath expr="//div[hasclass('col-lg-6')]" position="attributes">
+        <attribute name="class" add="col-lg-4 offset-lg-1" remove="col-lg-6" separator=" "/>
+    </xpath>
+    <xpath expr="//h2" position="replace" mode="inner">
+        <span class="text-break" style="font-size: 62px; font-weight: bolder;">Our R&amp;D Approach</span>
```

Comment:
Is this class comes from the editor? 🤔

Reply:
I have used this Bootstrap class in order to make headings' text break in smaller viewports. Otherwise the text overflows the container and generates an ugly horizontal margin. That could be also solved by using a `display-x` class.

Reply:
After further consideration, I think the less harmful solution would be keeping the `text-break` classes.

Ideally, we should have responsive user font sizes, but as I see here in this task https://www.odoo.com/web#action=333&active_id=974&cids=1&id=1958098&menu_id=4720&model=project.task&view_type=form this is going to take a while to be done.

In this case, I don't want to make those headings responsive, but rather forcing the text to wrap or break without creating an ugly overflow in small viewports:

(with `text-break`)
![image](https://user-images.githubusercontent.com/43962154/133752345-b1b6b7ba-6a9e-42f0-962b-6dcadbc92b3b.png)
![image](https://user-images.githubusercontent.com/43962154/133752403-fca50447-fe49-4587-92bc-4316b37904c3.png)

(without)
![image](https://user-images.githubusercontent.com/43962154/133752687-7ac4c678-2893-4154-a48f-7a390da4bce4.png)
![image](https://user-images.githubusercontent.com/43962154/133752711-e7439c2a-5ef2-4062-9e5f-438437c72d4d.png)

The class is only applied inside `font` tags and it disappears if the heading text is deleted and I don't see it to be harmful to other fonts which may face the same issue.

What do you think @qsm-odoo ?